motivaa-backend/motivaa-app/src/test/java/com/motivaa/boundary/UserControllerTest.java

```diff
@@ -86,11 +86,18 @@ void post_Users_MissingEmail() throws Exception {
             request.put("firstName", validFirstName);
             request.put("lastName", validLastName);
 
-            mockMvc.perform(post("/user-apis/users")
+            String responseJson =  mockMvc.perform(post("/user-apis/users")
                     .contentType(MediaType.APPLICATION_JSON)
                     .content(TestUtils.asJsonString(request))
                     .accept(MediaType.APPLICATION_JSON))
-                    .andExpect(status().isBadRequest());
+                    .andExpect(status().isBadRequest())
```

Comment:
This already checks if the response is 400. I don't understand why you assert for it later.

Reply:
the http code is part of the header and the body too, this checks the header, the assertion checks the body.

i think the real question is if its good practice to have it in the body too (its redundant information and generates the possibility of errors and confusion), but somehow I kinda like it being there (all the insurers I built integration always sent it in the body, I think I am used to it). 

ISCO also sends it and that pushed me over the line to do it here too. What is your view?